libraries/api/src/middleware/auth.js

```diff
@@ -319,11 +317,6 @@ const remoteAuthentication = ({signatureValidator, entry}) => {
         next();
       }
     } catch (err) {
-      if (err.code === 'AuthorizationError') {
-        return next(new ErrorReply({code: 'InsufficientScopes', message: err.messageTemplate, details: err.details}));
```

Comment:
I'm quite puzzled. The error that Simon reported lacked `details` field completely (or at least it wasn't showing up in the logs). Why would that be?

Reply:
The original bug was, indeed, right here -- we passed err.messageTemplate (which is undefined) instead of err.message (which contains the message simon expected).  The minimal fix to that bug was just messageTemplate -> message, but I took the chance to clean up some messiness in handling errors while I was at it.

Reply:
Yeah, it was a bit of leftovers from the refactor that @arshadkazmi42 worked on.